guide/index.md

```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
 );
 
-const { fishjamUrl, participantToken } = await response.json();
+const { url, peerToken } = await response.json();
 ```
 
-### Join Room and start streaming
+### 2. Join Room and start streaming
 
-To start streaming, you have to prepare your camera and join room:
+:::danger
 
-```ts
-import { joinRoom, useCamera } from "@fishjam-cloud/react-native-client";
+If you want to use a device camera, you must first request permission. Keep in mind that this won't work on iOS Simulator.
+Check [this `guide`](./react-native/installation#step-2-configure-app-permissions) for more information.
 
+:::
 
-function StartStreamingButton({ roomName, userName }) {
+To start streaming, you have to prepare your camera and join the room:
 
+```tsx
+function StartStreamingButton({ roomName, userName }) {
   const { prepareCamera } = useCamera();
 
-
   const startStreaming = useCallback(async () => {
     const response = await fetch(
-      `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?room=${roomName}&user=${userName}`
+      `https://cloud.fishjam.work/api/v1/connect/*YOUR_ID*/room-manager/${roomName}/users/${userName}`,
     );
-    const { fishjamUrl, participantToken } = await response.json();
+    const { url, peerToken } = await response.json();
 
-    await prepareCamera({ enableCamera: true });
+    await prepareCamera({ cameraEnabled: true });
 
-    await joinRoom(fishjamUrl, participantToken);
+    await joinRoom(url, peerToken);
+  }, []);
 
-  });
-
-  return <Button onPress={startStreaming}>Start Streaming</Button>
+  return <Button title={"Start Streaming"} onPress={startStreaming} />;
 }
 ```
 
-### Show other participants
+### 3. Show other peers
+
+:::note
+In order to get peers, you must first join a room. See steps above.
+:::
 
-Fetching other participants of your room, can be done with `useParticipants` hook. And to display their video stream,
+Fetching other peers in your room can be done with `usePeers` hook. To display their video stream,
 you can use `VideoRendererView` component. Example code could look like this:
 
-```ts
-import { useParticipants, VideoRendererView } from '@fishjam-cloud/react-native-client';
+```tsx
+function TracksView() {
+  const { peers } = usePeers();
+
+  const videoTracks = peers.flatMap((peer) =>
+    peer.tracks.filter((track) => track.type === "Video" && track.isActive),
+  );
+
+  return (
+    <View style={styles.tracksContainer}>
+      {videoTracks.map((track) => (
+        <VideoRendererView
+          style={styles.videoElement}
+          key={track.id}
+          trackId={track.id}
+          videoLayout="FIT"
+        />
+      ))}
+    </View>
+  );
+}
 
-function Component() {
-  const { participants } = useParticipants();
+const styles = StyleSheet.create({
+  tracksContainer: {
+    flex: 1,
+    alignItems: "center",
+  },
+  videoElement: {
+    width: "85%",
+    aspectRatio: 1,
+    backgroundColor: "gray",
+    marginVertical: 16,
+  },
+});
+```
+
+### Full example
```

Comment:
I'm not sure if this is needed. How am I suppose to use this example, when I try to integrate Fishjam Cloud with my app?

I think it would make more sense to link to our example app (or create some event simpler app that works more or less like the one below?)

Reply:
Imo this is needed. At least when I search packages to do stuff i see that provided ready example does not work i do not even bother to use it in my app.

Reply:
This is the only way to see everything in context and show the user how to use a minimal example. Personally I much prefer having one snippet that I can analyze and copy-paste. 

You can replace any of your app's screens and when you paste the example it will just work. Our example is too difficult to set up for a quick check.